app/ui/src/Pages/Project/panels/ProjectSettings/components/TabMembers/components/ManageMembers/ManageMembers.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  Button,
+  BUTTON_ALIGN,
+  ModalContainer,
+  ModalLayoutConfirmList,
+  Select,
+} from 'kwc';
+import React, { useRef } from 'react';
+import styles from './ManageMembers.module.scss';
+import useMembers from 'Graphql/hooks/useMembers';
+import { AccessLevel } from 'Graphql/types/globalTypes';
+import {
+  defaultModalInfo,
+  getModalInfo,
+  ModalInfo,
+} from '../../confirmationModals';
+import { GetProjectMembers_project_members } from 'Graphql/queries/types/GetProjectMembers';
+import MemberItem from './components/MemberItem/MemberItem';
+import DeleteIcon from '@material-ui/icons/Delete';
+import useBoolState from 'Hooks/useBoolState';
+
+const accessLevelSeparator = () => (
+  <Button
+    key="separator"
+    label="CHANGE MEMBERS LEVEL TO"
+    className={styles.separator}
+    align={BUTTON_ALIGN.LEFT}
+    disabled
+  />
+);
+
+type Props = {
+  projectId: string;
+  checkedMembers: GetProjectMembers_project_members[];
+  onCompleteRemove: () => void;
+};
+function ManageMembers({ projectId, checkedMembers, onCompleteRemove }: Props) {
+  const { toggle: toggleModal, value: isModalVisible } = useBoolState();
+  const modalInfo = useRef<ModalInfo>(defaultModalInfo);
+
+  const { updateMembersAccessLevel, removeMembersById } = useMembers(
+    projectId,
+    {
+      onCompleteUpdate: toggleModal,
+      onCompleteRemove: () => {
+        onCompleteRemove();
+        toggleModal();
```

Comment:
I think is more precise and safer to use closeModal function. We do not want to toggle from closed to opened after making these actions, I do not even know if this is possible.

```suggestion
      onCompleteUpdate: closeModal,
      onCompleteRemove: () => {
        onCompleteRemove();
        closeModal();
```